web-client/src/presenter/actions/broadcastIdleStatusActiveAction.test.ts

```diff
@@ -19,7 +20,7 @@ describe('broadcastIdleStatusActiveAction', () => {
     expect(
       applicationContext.getBroadcastGateway().postMessage.mock.calls[0][0],
     ).toMatchObject({
-      subject: 'idleStatusActive',
+      subject: LOGOUT_BROADCAST_MESSAGES.idleStatusActive,
```

Comment:
Nitpick only because of coverage benchmarks; the other branch of the ternary could use a test